<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowWindow</class>
 <widget class="QMainWindow" name="ShowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>运动员成绩查看界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color: rgb(68, 68, 68);
background-color: rgb(255, 255, 255);
border-color: rgb(68, 68, 68);
background-image: url(:/images/background.png);
background-repeat: no-repeat;  /* 防止图片重复 */
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/background.png);
background-repeat: no-repeat;  /* 防止图片重复 */
</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane{
	border: 2px solid rgb(210, 210, 210);
	background:rgb(246, 246, 246);
	border-top-color:transparent;
}
QTabWidget::tab-bar{
	background:rgb(0, 0, 0);
	subcontrol-position:left;
}
QTabBar::tab{
	width:216px;/*宽度根据实际需要进行调整*/
	height:32px;
	background:rgb(210, 210, 210);
	border: 2px solid rgb(210, 210, 210);
	border-top-left-radius: 8px;
	border-top-right-radius: 8px;
}
QTabBar::tab:selected{	
	background:rgb(246, 246, 246);
	border-bottom-color:rgb(246, 246, 246);
}
QTabBar::tab:!selected{
	background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(240, 240, 240, 255), stop:0.5 rgba(210, 210, 210, 255), stop:1 rgba(225, 225, 225, 255));
}


 </string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>32</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <widget class="QWidget" name="MAN_tab">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>🏃‍♂️‍➡️男性全马成绩表</string>
     </attribute>
     <widget class="QTableView" name="tableView_man">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="WOMAN_tab">
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>🏃‍♀️‍➡️女性半马成绩表</string>
     </attribute>
     <widget class="QTableView" name="tableView_woman">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_record">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menureturn">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>🔚 返  回</string>
    </property>
    <addaction name="actionHomePage"/>
   </widget>
   <widget class="QMenu" name="menufind">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>⚙️ 查  找</string>
    </property>
    <addaction name="actionfind"/>
   </widget>
   <widget class="QMenu" name="menurank">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>📜 查  看</string>
    </property>
    <addaction name="actionrankboard"/>
    <addaction name="actionold"/>
   </widget>
   <addaction name="menureturn"/>
   <addaction name="menufind"/>
   <addaction name="menurank"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHomePage"/>
   <addaction name="separator"/>
   <addaction name="actionfind"/>
   <addaction name="actionrankboard"/>
   <addaction name="actionold"/>
  </widget>
  <action name="actionfind">
   <property name="text">
    <string>🔎查找运动员</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionrankboard">
   <property name="text">
    <string>🏅运动员奖牌榜</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionHomePage">
   <property name="text">
    <string>🔙返回登录界面</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionold">
   <property name="text">
    <string>🧑‍🦳老年组成绩单</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
